### DISCUSSION TITLE
Svg pictures containing Chinese do not display text when rendered to docx.

### DISCUSSION THREAD

**Discussion by Dagwbl** [quarto-dev/quarto-cli#4547] [Q&A] [answered]
Can anyone tell me what is the reason or how to solve it?

eg: 

**original picture:**

![mermaid-server](https://user-images.githubusercontent.com/39078825/221494689-17946314-4c7c-4d7d-8120-3039aa3caf68.svg)

**rendered picture:**

![image](https://user-images.githubusercontent.com/39078825/221494747-4b8bd8ae-1251-4b58-9058-4a993dd1991a.png)

**Comment by Dagwbl** [comment]
it is fine while rendering to html format.

**Comment by baptiste** [comment, marked as answer]
Is this a mermaid diagram rendered by quarto, or generated by an external program and included as a picture? Looking at the SVG, I see some combination of `foreignObject` but also only `sans-serif` as font description, so presumably how text appears (or not) is heavily dependent on the rendering program.

```
<foreignObject width="96" height="25.59375" style="[...] font-family: sans-serif;"><div xmlns="http://www.w3.org/1999/xhtml" style=" [...] font-family: sans-serif; "><span class="nodeLabel" style="[...] font-family: sans-serif; ">执行业务逻辑</span></div></foreignObject>
```

**Reply by Dagwbl** [reply]
The current version of Mermaid in Quarto contains a problem. This is a picture that I rendered through external program (mermaid online) and then imported through Markdown syntax.

**Reply by baptiste** [reply]
Hmm, I suspect it's not an issue with the Chinese characters but more to do with foreignObject: it sounds similar to https://github.com/mermaid-js/mermaid-cli/issues/112

**Reply by baptiste** [reply]
adding `"htmlLabels": false` to the config as suggested in that thread appears to work, as far as I can tell, so you could give it a try

**Reply by Dagwbl** [reply]
Thank you! But I cannot find the config file to set the `"htmlLabels"`.

**Reply by Dagwbl** [reply]
is the `_quarto.yml`?

**Reply by baptiste** [reply]
![Screenshot 2023-02-27 at 20 39 01](https://user-images.githubusercontent.com/18970/221502661-b5d68070-b9c5-43c6-bdd2-3a5204f627aa.png)

**Reply by Dagwbl** [reply]
Oh, thank you, I will try it.
